WcaOnRails/db/migrate/20160522105428_create_delegate_reports.rb

```diff
@@ -0,0 +1,23 @@
+class CreateDelegateReports < ActiveRecord::Migration
+  def change
+    create_table :delegate_reports do |t|
+      t.string :competition_id
+
+      t.text :equipment
+      t.text :venue
+      t.text :organisation
+      t.string :schedule_url
+      t.text :incidents
+      t.text :remarks
+
+      t.string :discussion_url
+
+      t.datetime :posted_at
+
+      t.datetime :created_at
+      t.datetime :updated_at
+      t.timestamps null: false
```

Comment:
Why all these three fields are here? AFAIK `timestamps` should create both `created_at` and `updated_ed`